licenses.d2iq.yaml

```diff
@@ -143,6 +147,10 @@ resources:
     sources:
       - url: https://github.com/mesosphere/dkp-insights
         ref: main
+  - container_image: docker.io/mesosphere/insights:v0.5.0-beta.1
+    sources:
+      - url: https://github.com/mesosphere/dkp-insights
+        ref: main
```

Comment:
Here we should point to a specific tag. These is a replacement variable we can use to map container image tag to specific git tag by using.

```
        ref: ${image_tag}
```

We should update also the image mapping above `docker.io/mesosphere/insights-management:v0.5.0-beta.1` which points to the `main` as well.

Reply:
great. i'll update both, thank you